### ISSUE TITLE
How do I add ngx-formly/kendo with latest kendo libraries

### ISSUE THREAD

**Issue by luizfbicalho** [ngx-formly/ngx-formly#3571]
I'm creating a new project and I have problems to add the ngx-formly/kendo with the latest versions of kendo components.

```npm
npm ERR! Found: @progress/kendo-angular-inputs@10.1.2
npm ERR! node_modules/@progress/kendo-angular-inputs
npm ERR!   @progress/kendo-angular-inputs@"^10.1.2" from the root project
npm ERR!   peer @progress/kendo-angular-inputs@"9 - 10" from @progress/kendo-angular-treeview@7.1.5
npm ERR!   node_modules/@progress/kendo-angular-treeview
npm ERR!     @progress/kendo-angular-treeview@"7.1.5" from the root project
npm ERR!     peer @progress/kendo-angular-treeview@"^7.0.2" from @progress/kendo-angular-dropdowns@7.2.4
npm ERR!     node_modules/@progress/kendo-angular-dropdowns
npm ERR!       @progress/kendo-angular-dropdowns@"^7.2.4" from the root project
npm ERR!       1 more (@ngx-formly/kendo)
```

Shoudn't the ngx-formly/kendo  package be ok with any version above the versions in package.json?

**Comment by eshkard** [comment]
Hi,
When this commit will hit the npmjs?

Thank's in advance,
Shkard Evgeny

**Comment by aitboudad** [comment]
@eshkard I'll try to make a new release at the end of this week.

**Comment by eshkard** [comment]
Hi,
proposed change:
">=^4.0.0"
">=^7.0.2"
">=^9.0.3"
has a bit of uncommon syntax.

I'd propose revising at least to
">=4.0.0"
">=7.0.2"
">=9.0.3"
we are encountered ng error on the package upgrade regarding syntax (at least on angular 14)

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.1.2](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.1.2) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
